combine-an-array-into-a-string-using-the-join-method.js

```diff
@@ -0,0 +1,11 @@
+function sentensify(str) {
+  // Only change code below this line
+  var newStr = str;
+  var newArray = newStr.split(/\W/);
+  //console.log(newArray);
+  newStr = newArray.join(" ");
```

Comment:
and you can concatenate split and join, so you could just have 
```suggestion
  var newStr = str.split(/\W/).join(" ");
```

Reply:
Oh, that's compact!